operatorcourier/cli.py

```diff
@@ -0,0 +1,69 @@
+import argparse
+import sys
+from operatorcourier import api
+
+def main():
+    """Generate the CLI bits
+    """
+    parser = _CliParser()
+    parser.parse()
+
+class _CliParser():
+    """Class that generates the command line bits for the operator-courier cli tool
+    """
+
+    def __init__(self):
+        pass
+
+    def parse(self):
+        """Parse generates and evaluates the command level parser
+        """
+        parser = argparse.ArgumentParser(
+            description='Build, verify and push operator bundles into external app registry',
+            usage='''operator-courier <command> [<args>]
+
+These are the commands you can use:
+    verify      Create a bundle and test it for correctness.
+    push        Create a bundle, test it, and push it to an app registry.
+''')
+        parser.add_argument('command', help='Subcommand to run')
+
+        args = parser.parse_args(sys.argv[1:2])
+        if not hasattr(self, args.command):
+            parser.error('Unrecognized command')
+            parser.print_help()
+            exit(1)
+        
+        getattr(self, args.command)()
+
+    # Parse the verify command
+    def verify(self):
+        parser = argparse.ArgumentParser(description='Build and verify an operator bundle to test')
+        parser.add_argument('--source_dir', help='Path of your directory of yaml files to bundle. Either set this or use the files argument for bundle data.')
+        parser.add_argument('--files', help='Comma separated list of yaml files to bundle. Either set this or use the path argument for bundle data.')
+
+        args, leftovers = parser.parse_known_args(sys.argv[2:])
+        if args.files is None and args.source_dir is None:
```

Comment:
If only files or source_dir can be set, shouldn't we check if both are set? If this check isn't added, we may want to add the source_dir has greater priority